crates/worker/src/partition/state_machine/entries/get_invocation_output_command.rs

```diff
@@ -31,6 +31,10 @@ where
             response_sink: ServiceInvocationResponseSink::partition_processor(
                 self.invocation_id,
                 self.entry.completion_id,
+                self.invocation_status
+                    .get_invocation_metadata()
+                    .expect("Should be present when handling entries")
```

Comment:
Seems like you've copied code but haven't updated the expect message. Maybe check other places as well.